docs/theme/static/static.js

```diff
@@ -567,9 +567,9 @@ steal(
         })
 
 
-        $('#bithub-events-embed').bithubEmbed({
-            hubId: 5,
-            tenant_name: 'charming_volcano_7196_1'
+        $('#bithub-events-embed').upcomingEvents({
+            apiKey: 'AIzaSyBsNpdGbkTsqn1BCSPQrjO9OaMySjK5Sns',
```

Comment:
This API key is _not_ restricted right now but will be (by HTTP referrer) after I confirm that this is working on the live site.